webserver/Controllers/ClientController.cs

```diff
@@ -0,0 +1,149 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.EntityFrameworkCore;
+using webserver.Data;
+using webserver.Models;
+using webserver.Models.DTOs;
+using Newtonsoft.Json;
+
+namespace webserver.Controllers;
+
+[Authorize]
+[ApiController]
+[Route("api/v1/client")]
+public class ClientController : ControllerBase {
+
+    private readonly WebserverContext _context;
+    private readonly UserManager<Client> _userManager;
+    private readonly RoleManager<IdentityRole> _roleManager;
+
+    public ClientController(WebserverContext context, UserManager<Client> userManager, RoleManager<IdentityRole> roleManager){
+        _context = context;
+        _userManager = userManager;
+        _roleManager = roleManager;
+    }
+
+    [HttpGet("unique/{id}")]
```

Comment:
This can be just `/api/v1/client/231234`

```suggestion
    [HttpGet("/{id}")]
```

I don't see a reason for this `unique` prefix.

So, you can:
- list the clients with `GET /api/v1/client`
- add with `POST /api/v1/client`
- get with `GET /api/v1/client/{id}`
- edit with `PUT /api/v1/client/{id}`
- delete with `DELETE /api/v1/client/{}`

You can create all the CRUD with the exactly same structure ☝🏻 
https://restfulapi.net/resource-naming/

Reply:
I'll take a look before merging. I'm sure i got a "ambiguous endpoint" conflict for when reading a Specific Client Vs. List of CLients